vcd/resource_vcd_org_test.go

```diff
@@ -77,9 +80,20 @@ func testAccCheckOrgDestroy(s *terraform.State) error {
 			continue
 		}
 
-		org, err := govcd.GetOrgByName(conn.VCDClient, rs.Primary.Attributes["name"])
-		if org != (govcd.Org{}) || err != nil {
-			return fmt.Errorf("org with name %s was found", rs.Primary.Attributes["name"])
+		deleted := false
+		var org govcd.Org
+		var err error
+		for N := 0; N < 10; N++ {
```

Comment:
I would say this isn't sustainable for all systems. If we need to wait 100 count? Couldn't we use while?

Reply:
Right. I was actually fishing for comments on this one, as I have seen similar methods in a few places.
But we need to define a few public timeout values for long term operations and then use them consistently. I have a related task in my TODO list.

Reply:
ok